### PR TITLE
Refactoring EMIT examples for documentation

### PR DESCRIPTION
**Pull request by madelinecoleman1**

### CODE REVIEW

**Review by maxcapodi78 — APPROVED** [review]

File: examples/07-EMIT/ComputeInterferenceType.py.back [hunk 1]
```diff
@@ -1,3 +1,14 @@
+"""
+EMIT: Classify interference type
+--------------------------------
+This example shows how you can use PyAEDT to load and existing AEDT
```
Comment (MaxJPRey) [inline examples/07-EMIT/ComputeInterferenceType.py.back:4]:
```suggestion
This example shows how you can use PyAEDT to load an existing AEDT 
```

**Comment by jsalant22** [comment]
@maxcapodi78 Does it recursively do this or only for the top level files? If I have the directory structure below, which pages will it create html pages (and jupyter notebooks) for? Directories are in bold

- 07 - EMIT
  - Example 1.py
  - Example 2.py
  - Support Files
    -Support Function 1.py
    -Support Function 2.py
  - Example 3.py

Continuing the conversation here so we can make any necessary changes

**Comment by maxcapodi78** [comment]
> @maxcapodi78 Does it recursively do this or only for the top level files? If I have the directory structure below, which pages will it create html pages (and jupyter notebooks) for? Directories are in bold
> 
> * 07 - EMIT
>   
>   * Example 1.py
>   * Example 2.py
>   * Support Files
>     -Support Function 1.py
>     -Support Function 2.py
>   * Example 3.py
> 
> Continuing the conversation here so we can make any necessary changes

Not sure it's gonna work like this. In such specific case is more than a simple script example. I don't think Sphinx can handle it

File: examples/07-EMIT/interference_gui.py.back [hunk 2]
```diff
@@ -66,9 +78,14 @@ def __init__(self):
         super(MainWindow, self).__init__()
         self.setupUi(self)
         self.setup_widgets()
-    
-    def setup_widgets(self): # Widget definitions
 
+    ###############################################################################
+    # Setup widgets
+    # ~~~~~~~~~~~~~
+    # Define all widgets from the .ui file, connect the widgets to functions, define
```
Comment (PipKat) [inline examples/07-EMIT/interference_gui.py.back:85]:
```suggestion
    # Define all widgets from the UI file, connect the widgets to functions, define 
```

File: examples/07-EMIT/interference_gui.py.back [hunk 3]
```diff
@@ -233,8 +269,12 @@ def table_changed(self):
                 index = self.radio_dropdown.currentText()
             self.protection_levels[index] = values
 
+    ###############################################################################
+    # Save scenario matrix to as png file 
+    # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    # Save the scenario matrix table as a png file.
```
Comment (PipKat) [inline examples/07-EMIT/interference_gui.py.back:275]:
```suggestion
    # Save scenario matrix to as PNG file 
    # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    # Save the scenario matrix table as a PNG file.
```

File: examples/07-EMIT/interference_gui.py.back [hunk 4]
```diff
@@ -373,11 +431,21 @@ def populate_table(self):
...
+    # ~~~~~~~~~~~~~~~~~
+    # Close AEDT if the GUI is closed.
     def closeEvent(self, event):
         emitapp.close_desktop()
 
+###############################################################################
+# Run GUI
+# ~~~~~~~
+# Launch the GUI. Ensure that window.show() and app.exec_() are 
+# uncommented to run the GUI.
```
Comment (PipKat) [inline examples/07-EMIT/interference_gui.py.back:445]:
```suggestion
# Launch the GUI. If you want to run the GUI, uncomment the ``window.show()`` and
# ``app.exec_()`` method calls. 
```

Comment (jsalant22) [reply examples/07-EMIT/interference_gui.py.back]: @madelinecoleman1 It sounds like we need to move interference_classification.py out of the examples directory. We can move it to pyaedt\emit_core for now at least. Do you want to make that change with this PR or the testing PR? (either works for me)

Comment (madelinecoleman1) [reply examples/07-EMIT/interference_gui.py.back]: @jsalant22 sounds good, I'll move that file. I'll do it in this PR

**Comment by jsalant22** [comment]
@madelinecoleman1 It sounds like we need to move interference_classification.py out of the examples directory. We can move it to pyaedt\emit_core for now at least. Do you want to make that change with this PR or the testing PR? (either works for me)

**Review by jsalant22 — APPROVED** [review]

**Comment by jsalant22** [comment]
@madelinecoleman1 Looks like this one has a merge conflict.

**Comment by madelinecoleman1** [comment]
@jsalant22, fixed the merge conflicts, should be good now

**Comment by jsalant22** [comment]
@madelinecoleman1 Sorry, but the auto checks found a few other "issues". Largely just cleaning up some code. Here's a link to the issues: https://app.codacy.com/gh/ansys/pyaedt/pullRequest?prid=12176027

File: pyaedt/emit_core/interference_classification.py [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+            rx_band_objects = radios[rx_radio].bands()
+            if tx_radio == rx_radio:
+                # skip self-interaction
+                rx_powers.append("N/A")
+                rx_colors.append("white")
+                continue
+
+            max_power = -200
+            tx_bands = rev.get_band_names(tx_radio, modeTx)
+            tx_band_objects = radios[tx_radio].bands()
```
Comment (jsalant22) [inline pyaedt/emit_core/interference_classification.py]:
```suggestion

```

File: pyaedt/emit_core/interference_classification.py [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+                rx_freq = rev.get_active_frequencies(rx_radio, rx_band, modeRx)[0]
+
+                # The start and stop frequencies define the Band's extents,
+                # while the active frequencies are a subset of the Band's frequencies
+                # being used for this specific project as defined in the Radio's Sampling.
+                rx_start_freq = radios[rx_radio].band_start_frequency(rx_band_objects[i])
+                rx_stop_freq = radios[rx_radio].band_stop_frequency(rx_band_objects[i])
+                rx_channel_bandwidth = radios[rx_radio].band_channel_bandwidth(rx_band_objects[i])
+
+                for j, tx_band in enumerate(tx_bands):
```
Comment (jsalant22) [inline pyaedt/emit_core/interference_classification.py]:
```suggestion
                for tx_band in tx_bands:
```

**Pull request merged**